### DISCUSSION TITLE
Cannot join or create a playlist on Mobile.

### DISCUSSION THREAD

**Discussion by yesseruser** [ppy/osu#21190] [Q&A] [answered]
### Type

Game behaviour

### Bug description

When I try to join or create a playlist, the GUI disappears and a notification appears saying "An unhandled error occured".

I have got a Google Pixel 6 with Android 13 

### Screenshots or videos

https://youtu.be/wJ9aH-Wf5mk
I can't upload the video here so I uploaded it to YouTube.

### Version

2022.1101.0-lazer

### Logs

I am very sorry, I can't access the Android/Data folder on Android 13 because the access is denied to file managers and I don't have access to PC.

**Comment by frenzibyte** [comment]
Logs are essential, we cannot investigate this issue as-is. Converting to discussion thread until logs are provided.

**Comment by yesseruser** [comment]
I managed to get an app to access the logs, here they are:
[database.log](https://github.com/ppy/osu/files/10147153/database.log)
[network.log](https://github.com/ppy/osu/files/10147154/network.log)
[performance.log](https://github.com/ppy/osu/files/10147155/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10147156/runtime.log)

**Reply by bdach** [reply]
All of those files are 0 bytes. As in empty.

**Reply by yesseruser** [reply]
How is that possible? I'm sure they had contains when I uploaded them.

**Reply by yesseruser** [reply, marked as answer]
The bug has been fixed, I don't know which update was it.
